packages/foreman/rubygem-rubyipmi/rubygem-rubyipmi.spec

```diff
@@ -1,80 +1,91 @@
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name rubyipmi
 
-Summary: A ruby wrapper for ipmi command line tools
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.10.0
-Release: 3%{?dist}
+Release: 4%{?dist}
+Summary: A ruby wrapper for ipmi command line tools that supports ipmitool and freeipmi
 Group: Development/Languages
-License: LGPLv2
-URL: https://github.com/logicminds/rubyipmi
+License: LGPLv2.1
+URL: http://github.com/logicminds/rubyipmi
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
-%if 0%{?fedora} > 18 || 0%{?rhel} >= 7
+
+# start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
-%else
-Requires: %{?scl_prefix_ruby}ruby(abi)
-%endif
+Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: ipmitool
```

Comment:
as far as I understand the gem, it's a wrapper around `ipmitool` and `freeipmi`, so I think the requires should remain (outside of the generated block)